packages/constellation/src/components/Compositions/Logo/Logo.tsx

```diff
@@ -0,0 +1,60 @@
+import cx from 'classnames'
+import React from 'react'
+import { Textfit } from 'react-textfit'
+
+import { InternalLogoComponent, LogoComponent } from './Logo.types'
+
+const InternalLogo: InternalLogoComponent = ({ circle, className, value }) => {
+  return (
+    <div
+      className={cx(
+        { '!rounded-full': circle },
+        'w-8 h-8 rounded flex justify-center items-center bg-primary relative overflow-hidden',
+        className,
+      )}
+    >
```

Comment:
It seems like there may be a need for transparency with images as primary logos. Maybe I'm misunderstanding what these mocks are getting at: https://www.figma.com/file/lI692dBvIP9Hmx2uCH0cnI/03---Web-%7C-Shared?node-id=4404%3A19240

Is there a way to account for that here?

Reply:
Ah I thought that was just the image in the background but a transparent background makes sense. I can change it to only have a solid background when the value is text. Or the other option would be to just add a background colour prop.

Thoughts?

Reply:
Hmm yea, an optional background color prop might be handy here. Might be the best option without knowing all the use cases upfront.